glTF-BinExporter/glTF/GlTFRootModel.cs

```diff
@@ -145,6 +146,9 @@ public void SerializeToGLB(in MemoryStream memoryStream)
                     bv.byteOffset = bv.bufferRef.binaryOffset;
                 }
 
+                //Have to get rid of the other buffers so only the new buffer
+                //all the data was dumped into remains
+                buffers.Clear();
```

Comment:
Man... how did that even work before? 👍  Thanks!

While reviewing this I realize I may have misunderstood the spec initially: I now think it is valid to create multiple binary buffers, instead of mashing everything into one and using views to discern the data blobs. Not sure what the implications are of that yet. The validator fails with:
```
           {
                "code": "ACCESSOR_TOTAL_OFFSET_ALIGNMENT",
                "message": "Accessor's total byteOffset 397965 isn't a multiple of componentType length 4.",
                "severity": 0,
                "pointer": "/accessors/0"
            },
```
Maybe that's a side effect of this weird choice. Will create a separate ticket for that though.

Reply:
Yes, I think this changed. While we probably can cram multiple things into a single buffer it's unnecessarily complicated and makes the code and final output file confusing. The validator error makes sense to me since we're probably mixing floats/bytes/integers all in one buffer and just fiddling with offsets. It's not super imperative though so we'll get to #22 when either of us has a chance.